app/Http/Controllers/PrintController.php

```diff
@@ -91,6 +91,13 @@ public function print(Request $request) {
             return back()->withErrors(['print' => __('print.error_printing')]);
         }
     }
+
+    public function sumOfActiveFreePages() {
```

Comment:
I just realized that we are in the controller here, so you would have to pass the values like you do above to the blade template. A more elegant and logical solution would be is to move the logic to the models.
You would create a `sumOfActive()` function in FreePages doing the same thing, but with `$this`.
That way you could directly access them, without passing, like this: Auth::user()->freePages->sumOfActive().